modules/ossm-config-openshift-monitoring-only.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/metrics/ossm-metrics-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-only_{context}"]
+= Configuring OpenShift Monitoring with Service Mesh
+:context: ossm-config-openshift-monitoring-only
+
+//Need some kind of intro
+//Possible file name may change
+//Possible assembly file may change
+//Assemblies, topic map info needs to be worked out still for 3.0, now that docs has a better idea of what is to be included for 3.0 TP1.
+
+//FIX THE STYLE THINGS ONCE DEV HAS REVIEWED/APPROVED THE CONTENT. First priority is ensuring the steps are in the correct order, and are accurate for 3.0 TP1.
+
+The following steps show how to integrate {SMProductName} with user-workload monitoring.
+
+.Prerequisite
+
+ * Monitoring for user-defined projects is enabled. See "Enabling monitoring for user-defined projects".
+
+.Procedure
+
+. Apply a `ServiceMonitor` object to monitor the Istio control plane:
++
+.Example `ServiceMonitor` object
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: istiod-monitor
+  namespace: istio-system
+spec:
+  targetLabels:
+  - app
+  selector:
+    matchLabels:
+      istio: pilot
+  endpoints:
+  - port: http-monitoring
+    interval: 30s
+    relabelings:
+    - action: replace
+      replacement: "basic-istio-system"
+      targetLabel: mesh_id
+----
+
+. Apply a `PodMonitor` object to collect metrics from Istio proxies:
++
+.Example PodMonitor object
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: istio-proxies-monitor
+  namespace: istio-system
+spec:
+  selector:
+    matchExpressions:
+    - key: istio-prometheus-ignore
+      operator: DoesNotExist
+  podMetricsEndpoints:
+  - path: /stats/prometheus
+    interval: 30s
+    relabelings:
+    - action: keep
+      sourceLabels: [__meta_kubernetes_pod_container_name]
+      regex: "istio-proxy"
+    - action: keep
+      sourceLabels: [__meta_kubernetes_pod_annotationpresent_prometheus_io_scrape]
+    - action: replace
+      regex: (\d+);(([A-Fa-f0-9]{1,4}::?){1,7}[A-Fa-f0-9]{1,4})
+      replacement: '[$2]:$1'
+      sourceLabels: [__meta_kubernetes_pod_annotation_prometheus_io_port,
+      __meta_kubernetes_pod_ip]
+      targetLabel: __address__
+    - action: replace
+      regex: (\d+);((([0-9]+?)(\.|$)){4})
+      replacement: $2:$1
```

Comment:
When I go to apply this YAML `kubectl` or `oc` complain about this line. I _think_ this is equivalent 
```suggestion
      replacement: '$2:$1'
```